### DISCUSSION TITLE
QAT for YOLOv8

### DISCUSSION THREAD

**Discussion by YaKalmar0** [openvinotoolkit/nncf#2165] [Q&A]
Hello there! Is there any out-of-the-box technique to use in the scope of Quantization Aware Training for YOLOv8 model?
Could only find a notebook demonstrating PTQ for this type of models, but nothing about QAT.

**Comment by alexsu52** [comment]
Hello,

NNCF has several examples of using QAT from object detection domain https://github.com/openvinotoolkit/nncf/tree/develop/examples/torch/object_detection. You can use these examples as a basis for integrating the power of NNCF QAT into your own YOLOv8 training pipeline.

Why did you decide to use QAT for the YOLOv8 model? Did you not achieve the desired accuracy for the quantized model using PTQ? Have you tried nncf.quantize_with_accuracy_control(...)?

**Reply by YaKalmar0** [reply]
Thanks for your reply! Will take the example you sent into work.
In my research I am particularly interested in performing QAT not only from the perspective of accuracy results, but also from the perspective of power consumption. YOLOv8 is just a model on the list to check.
